src/cli/telemetry/test/telemetry.spec.ts

```diff
@@ -185,101 +181,39 @@ describe('prepareData', () => {
     });
   });
 
-  it('updates when there is a PWA config', async () => {
-    const config: d.ValidatedConfig = {
-      flags: createConfigFlags(),
-      logger: mockLogger(),
-      outputTargets: [{ type: 'www', baseUrl: 'https://example.com', serviceWorker: { swDest: './tmp' } }],
-    };
+  describe('has_app_pwa_config property', () => {
+    it('sets `has_app_pwa_config` to true when there is a service worker', async () => {
+      const config = mockValidatedConfig({
+        outputTargets: [{ type: 'www', baseUrl: 'https://example.com' }],
+      });
 
-    const data = await telemetry.prepareData(coreCompiler, config, sys, 1000);
+      const data = await telemetry.prepareData(coreCompiler, config, sys, 1000);
 
-    expect(data).toEqual({
-      arguments: [],
-      build: coreCompiler.buildId,
-      component_count: undefined,
-      config: {
-        flags: {
-          args: [],
-          knownArgs: [],
-          task: null,
-          unknownArgs: [],
-        },
-        outputTargets: [
-          {
-            baseUrl: 'omitted',
-            serviceWorker: {
-              swDest: 'omitted',
-            },
-            type: 'www',
-          },
-        ],
-      },
-      cpu_model: '',
-      duration_ms: 1000,
-      has_app_pwa_config: true,
-      os_name: '',
-      os_version: '',
-      packages: [],
-      packages_no_versions: [],
-      rollup: coreCompiler.versions.rollup,
-      stencil: coreCompiler.versions.stencil,
-      system: 'in-memory __VERSION:STENCIL__',
-      system_major: 'in-memory __VERSION:STENCIL__',
-      targets: ['www'],
-      task: null,
-      typescript: coreCompiler.versions.typescript,
-      yarn: false,
+      expect(data.has_app_pwa_config).toBe(true);
```

Comment:
I don't think we need to test the entire object here. The single property should suffice (IMO)